doc/lsst.ap.pipe/pipeline-bps.rst

```diff
@@ -0,0 +1,141 @@
+    pipetaskInit:
+      runQuantumCommand: '${CTRL_MPEXEC_DIR}/bin/pipetask --long-log run -b {butlerConfig} -i {inCollection} --output {output} --output-run {outCollection} --init-only --skip-existing --register-dataset-types --qgraph {qgraphFile} --clobber-partial-outputs --no-versions'
+    # Option to list other pipeline tasks being run here with default overrides
+    imageDifference:
+      requestMemory: 4096
+  
+  # Condor backend stuff
+  wmsServiceClass: lsst.ctrl.bps.wms.htcondor.htcondor_service.HTCondorService
+  clusterAlgorithm: lsst.ctrl.bps.quantum_clustering_funcs.single_quantum_clustering
+  
+  # Create quantum graph for use by all tasks
+  createQuantumGraph: '${CTRL_MPEXEC_DIR}/bin/pipetask qgraph -d "{dataQuery}" -b {butlerConfig} -i {inCollection} -p {pipelineYaml} -q {qgraphFile} --qgraph-dot {qgraphFile}.dot'
+  
+  # Main pipetask run call
+  runQuantumCommand: '${CTRL_MPEXEC_DIR}/bin/pipetask --long-log run -b {butlerConfig} -i {inCollection} --output {output} --output-run {outCollection} --extend-run --skip-init-writes --qgraph {qgraphFile} --clobber-partial-outputs --no-versions'
+
+Notes on the yaml file
+----------------------
+
+* The `computeSite` option determines where your jobs will run. As of now (April 2021) the typical choice will be `ncsapool`. Other options may be possible and/or make more sense for you, depending on what you're doing. It is best to ask the bps expoerts about that.
+* Note that you must use the long option names in a yaml file for the apparopriate pipetask options, e.g. `butlerConfig` instead of `-i`, `dataQuery` instead of `-d`, etc.
+* You can request default resource requirements such as memory or run time at the top level of the yaml (see the `requestMemory` line above), but you can give other values for specific task types if you want (see the higher requestMemory value in the imageDifference section under pipetask).
+* Don't forget to set your butler, input and output collections, and any other absolute paths according to your own work area.
+
+.. _section-ap-pipe-pipeline-bps-submit:
+
+Submit and Monitor
+==================
+
+Now we should be able to run a `bps submit` command with an appropriate yaml file:
+
+.. prompt:: bash
+
+   bps submit yaml/bps_ApPipe.yaml
+
+To see the status of our submission we can do
+
+.. prompt:: bash
+
+   bps report
+
+Which will look something like::
+
+  X     STATE  %S       ID OPERATOR   PRJ      CMPGN                     PAYLOAD                        RUN                                               
+  -----------------------------------------------------------------------------------------------------------------------
+  F    RUNNING  83    25639 kherner    ApPipe kh_default_bestSeeing_FULL ApPipe_default_bestSeeing_FULL u_kherner_ApPipe_default_bestSeeing_FULL_20210329T
+
+You can get additional information about the status of your run by passing the ``--id IDNUM`` option to ``bps report``. For example: ::
```

Comment:
I think three colons is too many colons. And you should probably have a `.. prompt:: bash` here.
``